components/authz-service/storage/postgres/migration/iam.go

```diff
@@ -0,0 +1,125 @@
+package migration
```

Comment:
Nope! This code will only exist here. It will only be guaranteed to work against schema version 74. And unless something goes very wrong, no one should every modify this code again.